libs/damap/src/lib/components/dmp/project/manual-project-input/manual-project-input.component.html

```diff
@@ -18,6 +18,7 @@
     [control]="description"
     label="dmp.steps.project.input.description"></app-textarea-wrapper>
   <button
+    style="margin-left: 5px"
```

Comment:
can we set this on the button element throughout the application? So we do not have to add this everywhere